webapp/pages/profile/_id/_slug.vue

```diff
@@ -239,11 +241,16 @@
           <ds-flex-item class="profile-top-navigation">
             <ds-card class="ds-tab-nav">
               <ds-flex>
-                <ds-flex-item class="ds-tab-nav-item ds-tab-nav-item-active">
+
+                <ds-flex-item
+                    v-tooltip="{content: 'Your Posts', placement: 'right', delay: { show: 500 }}"
```

Comment:
No untranslated strings, please.

Reply:
it should be everything in english. in my local version there is nothing more german.i pushed the latest one

Reply:
![gif](http://diestelkind.de/github_img/ezgif.com-video-to-gif.gif "Logo Title Text 1")